helloextend-protection/includes/class-helloextend-protection-orders.php

```
@@ -77,16 +77,46 @@ public function __construct($helloextend_protection, $version)
+        }
+
+        // Retrieve URL safely: prefer wp_get_attachment_image_url() on WP ≥4.4
+        if (function_exists('wp_get_attachment_image_url')) {
+            $url = wp_get_attachment_image_url($image_id, 'full');
+        } else {
+            $src = wp_get_attachment_image_src($image_id, 'full');
+            $url = (is_array($src) && isset($src[0])) ? $src[0] : null;
+        }
+
+        return $url ?: null;
+    }
```

PHPMD (2.15.0)

The method get_product_image_url() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (Code Size Rules)

(CyclomaticComplexity)